### ISSUE TITLE
pip

### ISSUE THREAD

**Issue by nickl-** [Aeronautics/aero#9]
EDIT: Find out what PIP is as mentioned in the README

= What the? 

What is this doing on the list? Does this mean we will be having ruby, perl, erlang, node.js, python et al as well?

No current implementation

This task needs to be completed as part of milestone Version 0.1 see #2

**Comment by alganet** [comment]
I don't understood this issue =S

**Comment by nickl-** [comment]
My mistake I see it is PIP not sure what that is neither but at least it makes better sense

**Comment by nickl-** [comment]
[PIP](http://www.pip-installer.org/en/latest/index.html) and what was that PyPi sounds nasty.

**Comment by alganet** [comment]
PyPi is actually very awesome, @jaysonsantos wrote a protocol binary compatible Memcache server in Python running on it =D

**Comment by jaysonsantos** [comment]
@alganet pypi is not pypy it is a package index

**Comment by alganet** [comment]
That explains why I changed my python skills to 0 on that table in another
issue :P

**Comment by nickl-** [comment]
XD

**Comment by nickl-** [comment]
So we found pip we found pypi we've gotten pythonic with generators and all sorts of funk. Dove into the deep end and python quickly came alive so to speak. Really glad we chose this route Tx @jaysonsantos !!!

Pip enabled with commands 
- search
- install

It does not appear to have any means to query more info nor does it have any uninstall capabilities. Something for other issues closing this thread.
